hack/uninstall-operator.sh

```diff
@@ -87,11 +116,21 @@ main() {
 		fail "failed to parse args"
 		return 1
 	}
+	$SHOW_HELP && {
+		print_usage
+		return 0
+	}
+
+	[[ -z "$OPERATOR_VERSION" ]] && {
+		info "No operator version specified; finding the installed version"
+		find_installed_version "$operator"
+		ok "Found - Kepler Operator version: $OPERATOR_VERSION"
+	}
 
 	header "Resources of Kepler Operator - $OPERATOR_VERSION"
```

Comment:
add a heavy line below printing CSV.

Reply:
I will make it a header

Reply:
Also notices that the `operators` is lingering which I have now deleted.